src/main/java/edu/ucsb/mapache/models/SlackSlashCommandParams.java

```diff
@@ -0,0 +1,146 @@
+
+    public String getChannelName() {
+        return this.channelName;
+    }
+
+    public void setChannelName(String channelName) {
+        this.channelName = channelName;
+    }
+
+    public String getUserId() {
+        return this.userId;
+    }
+
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    public String getUserName() {
+        return this.userName;
+    }
+
+    public void setUserName(String userName) {
+        this.userName = userName;
+    }
+
+    public String getCommand() {
+        return this.command;
+    }
+
+    public void setCommand(String command) {
+        this.command = command;
+    }
+
+    public String getText() {
+        return this.text;
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+
+    public String getResponseUrl() {
+        return this.responseUrl;
+    }
+
+    public void setResponseUrl(String responseUrl) {
+        this.responseUrl = responseUrl;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o == this)
+            return true;
+        if (!(o instanceof SlackSlashCommandParams)) {
+            return false;
+        }
+        SlackSlashCommandParams SlackSlashCommandParams = (SlackSlashCommandParams) o;
+        EqualsBuilder builder = new EqualsBuilder();
+        
+        builder.append(token, SlackSlashCommandParams.token).append(teamId, SlackSlashCommandParams.teamId)
+                .append(teamDomain, SlackSlashCommandParams.teamDomain)
+                .append(channelId, SlackSlashCommandParams.channelId)
+                .append(channelName, SlackSlashCommandParams.channelName)
+                .append(userId, SlackSlashCommandParams.userId)
+                .append(userName, SlackSlashCommandParams.userName)
+                .append(command, SlackSlashCommandParams.command).append(text, SlackSlashCommandParams.text)
+                .append(responseUrl, SlackSlashCommandParams.responseUrl);
+                
+        return builder.isEquals();
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(token, teamId, teamDomain, channelId, channelName, userId, userName, command, text,
+                responseUrl);
+    }
+
+    @Override
+    public String toString() {
+        return "{" + " token='" + getToken() + "'" + ", teamId='" + getTeamId() + "'" + ", teamDomain='"
+                + getTeamDomain() + "'" + ", channelId='" + getChannelId() + "'" + ", channelName='" + getChannelName()
+                + "'" + ", userId='" + getUserId() + "'" + ", userName='" + getUserName() + "'" + ", command='"
+                + getCommand() + "'" + ", text='" + getText() + "'" + ", responseUrl='" + getResponseUrl() + "'" + "}";
+    }
+
+    // For explanation of "[\\s\\p{Z}]" see: https://stackoverflow.com/a/26713907
+    public String[] getTextParts() {
+        // String textParts[]= this.text.split("[\\s\\p{Z}]+"); <------- btkh4 advice was enacted below
```

Comment:
It isn't necessary to keep old code or to have comments mentioning that you changed something in response to a code review.

Reply:
Done, deleted the comments